TCMSApp/src/client/app/core/constants.js

```diff
@@ -6,5 +6,40 @@
         .module('app.core')
         .constant('moment', moment)
         .constant('faker', faker)
-        .constant('Trello', Trello)
+        .constant('filterFields', filterFields())
+
+    function filterFields(){
```

Comment:
if we will have some API  constants here too this file will be too crowded. your filter settings deserve the separate file.